src/Biconomy.js

```diff
@@ -630,17 +634,21 @@ async function sendSignedTransaction(engine, payload, end) {
             forwardedData = decodedTx.data;
 
             let paramArrayForGasCalculation = [];
+            let typeString = "";
             for (let i = 0; i < params.length; i++) {
               paramArrayForGasCalculation.push(
                 _getParamValue(params[i])
               );
+              typeString = typeString + params[i].type.toString()+",";
             }
+            typeString = typeString.substring(0,typeString.length - 1);
```

Comment:
What this will return if params.length = 0 ?

Reply:
this should be neough

`if(params.length > 0) {
        typeString = typeString.substring(0,typeString.length - 1);
        }`